qdrant-landing/content/articles/sparse-vectors.md

```diff
@@ -0,0 +1,396 @@
+| **Information Density**   | Less dense, focuses on key features | Highly dense, capturing nuanced relationships |
+| **Example Applications**  | Recommendation systems | RAG, many general machine learning tasks |
+
+Where do Sparse Vectors fail though? They're not great at capturing nuanced relationships between words. For example, they can't capture the relationship between "king" and "queen" as well as dense vectors.
+
+# SPLADE
+
+Let's check out [SPLADE](https://europe.naverlabs.com/research/computer-science/splade-a-sparse-bi-encoder-bert-based-model-achieves-effective-and-efficient-full-text-document-ranking/?utm_source=qdrant&utm_medium=website&utm_campaign=sparse-vectors&utm_content=article&utm_term=sparse-vectors), an excellent way to make sparse vectors. Let's look at some numbers first. Higher is better:
+
+| Model              | MRR@10 (MS MARCO Dev) | Type           |
+|--------------------|---------|----------------|
+| BM25               | 0.184   | Sparse |
+| TCT-ColBERT        | 0.359   | Dense  | 
+| doc2query-T5 [link](https://github.com/castorini/docTTTTTquery)  | 0.277   | Sparse |
+| SPLADE             | 0.322   | Sparse |
+| SPLADE-max         | 0.340   | Sparse |
+| SPLADE-doc         | 0.322   | Sparse |
+| DistilSPLADE-max   | 0.368   | Sparse |
+
+All numbers are from [SPLADEv2](https://arxiv.org/abs/2109.10086). MRR is [Mean Reciprocal Rank](https://www.wikiwand.com/en/Mean_reciprocal_rank#References), a standard metric for ranking. [MS MARCO](https://microsoft.github.io/MSMARCO-Passage-Ranking/?utm_source=qdrant&utm_medium=website&utm_campaign=sparse-vectors&utm_content=article&utm_term=sparse-vectors) is a dataset for evaluating ranking and retrieval for passages.  
+
+SPLADE is quite flexible as a method, with regularization knobs that can be tuned to obtain [different models](https://github.com/naver/splade) as well: 
+
+> SPLADE is more a class of models rather than a model per se: depending on the regularization magnitude, we can obtain different models (from very sparse to models doing intense query/doc expansion) with different properties and performance. 
+
+First, let's look at how to create a sparse vector. Then, we'll look at the concepts behind SPLADE.
+
+# Creating a Sparse Vector
+
+We'll explore two different ways to create a sparse vector. The higher performance way to create a sparse vector from dedicated document and query encoders. We'll look at a simpler approach -- here we will use the same model for both document and query. We will get a dictionary of token ids and their corresponding weights for a sample text - representing a document.
+
+If you'd like to follow along, here's a [Colab Notebook](https://colab.research.google.com/gist/NirantK/ad658be3abefc09b17ce29f45255e14e/splade-single-encoder.ipynb), [alternate link](https://gist.github.com/NirantK/ad658be3abefc09b17ce29f45255e14e) with all the code.
```

Comment:
How about putting the notebook as another example [here](https://github.com/qdrant/examples/) and putting a link to https://githubtocolab.com?